modules/RTC/RTCUtils.js

```diff
@@ -93,6 +93,9 @@ let disableAGC = false;
 // Disables Highpass Filter
 let disableHPF = false;
```

Comment:
I think disableHPF is no longer used anywhere, is it? Please get rid off it in that case :-)